microcosm_logging/factories.py

```diff
@@ -10,6 +10,7 @@
 
 @defaults(
     default_format="{asctime} - {name} - [{levelname}] - {message}",
+    json_required_keys="%(asctime)s - %(name)s - %(filename)s - %(levelname)s - %(levelno) - %(message)s",
```

Comment:
This is a little magical. We're using the [PythonJsonLogger](https://github.com/madzak/python-json-logger) which extends the python logging.formatter which takes an old style format string as an argument to its constructor. 

The PythonJsonLogger keeps this api, but parses the `fmt` argument (which is an old-style format string) to get the set of require keys for the json log that it will generate. 

That means, to ensure we get keys like `levelname` we need to pass a format string that mentions those keys and the PythonJSONLogger will try to convert the old style of specifying require keys (i.e. via a format string) and convert it into keys that it will make sure show up in the json it generates.